SPECS/kernel/config_aarch64

```diff
@@ -398,6 +398,7 @@ CONFIG_ARM64_ERRATUM_2645198=y
 CONFIG_ARM64_WORKAROUND_SPECULATIVE_UNPRIV_LOAD=y
 CONFIG_ARM64_ERRATUM_2966298=y
 CONFIG_ARM64_ERRATUM_3117295=y
+CONFIG_ARM64_ERRATUM_3194386=y
```

Comment:
result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/93696d8f96a98aa2e1d98bf0003f83b2321090cf